android_jni/avifandroidjni/src/androidTest/java/org/aomedia/avif/android/StillImageTest.java

```diff
@@ -25,18 +20,33 @@
 @RunWith(Parameterized.class)
 public class StillImageTest {
 
+  private static final Image[] IMAGES = {
+    // Parameter ordering: filename, width, height, depth, alphaPresent, threads.
+    new Image("fox.profile0.10bpc.yuv420.avif", 1204, 800, 10, false, 1),
+    new Image("fox.profile0.10bpc.yuv420.monochrome.avif", 1204, 800, 10, false, 1),
+    new Image("fox.profile0.8bpc.yuv420.avif", 1204, 800, 8, false, 1),
+    new Image("fox.profile0.8bpc.yuv420.monochrome.avif", 1204, 800, 8, false, 1),
+    new Image("fox.profile1.10bpc.yuv444.avif", 1204, 800, 10, false, 1),
+    new Image("fox.profile1.8bpc.yuv444.avif", 1204, 800, 8, false, 1),
+    new Image("fox.profile2.10bpc.yuv422.avif", 1204, 800, 10, false, 1),
+    new Image("fox.profile2.12bpc.yuv420.avif", 1204, 800, 12, false, 1),
+    new Image("fox.profile2.12bpc.yuv420.monochrome.avif", 1204, 800, 12, false, 1),
+    new Image("fox.profile2.12bpc.yuv422.avif", 1204, 800, 12, false, 1),
+    new Image("fox.profile2.12bpc.yuv444.avif", 1204, 800, 12, false, 1),
+    new Image("fox.profile2.8bpc.yuv422.avif", 1204, 800, 8, false, 1)
```

Comment:
looks like the last two parameters are always `false, 1`. Any plans to add further coverage?

Reply:
yes, these tests are a bit slow sometimes on emualtors. so i will probably replace some of these files with files that have alpha and change the thread count for some of the existing ones in a follow-up.